x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_page/alerts_page.tsx

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useState, useCallback, useEffect } from 'react';
+import { get } from 'lodash';
+import {
+  EuiDataGridCellValueElementProps,
+  EuiDataGridControlColumn,
+  EuiFlexItem,
+  EuiFlexGroup,
+} from '@elastic/eui';
+import * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import { AlertConsumers } from '@kbn/rule-data-utils';
+import {
+  RuleRegistrySearchRequest,
+  RuleRegistrySearchResponse,
+  RuleRegistrySearchRequestPagination,
+} from '../../../../../../rule_registry/common';
+import { AlertsTable } from '../alerts_table';
+import { useKibana } from '../../../../common/lib/kibana';
+import { AbortError } from '../../../../../../../../src/plugins/kibana_utils/common';
+import { AlertsData } from '../../../../types';
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface Props {}
+
+const consumers = [
+  AlertConsumers.APM,
+  AlertConsumers.LOGS,
+  AlertConsumers.UPTIME,
+  AlertConsumers.INFRASTRUCTURE,
+];
+
+const defaultPagination = {
+  pageSize: 10,
+  pageIndex: 0,
+};
+
+const AlertsPage: React.FunctionComponent<Props> = (props: Props) => {
```

Comment:
I know this is a dummy page, but if you don't need props, you can simply not accept a parameter here.

Reply:
will do